React/src/components/FormControl/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useForm } from 'react-hook-form';
+
+import { RegisterUser } from 'interfaces/RegisterUser';
+
+import styles from './styles.module.scss';
+
+export function FormControl() {
+  const { register, handleSubmit } = useForm<RegisterUser>();
+
+  return (
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
+        <form
+          onSubmit={handleSubmit((data) => console.log({ users: data }))}
```

Comment:
Little typo here. Also keep in mind that here we are returning a console.log(). It will work as expected, but we need to more declarative with our intent. Because in this case we would be returning and executing "console.log()" and we only want to execute it.

```suggestion
  onSubmit={handleSubmit((data) => {
    console.log({ user: data })
  }
)}
```